lib/TaskRouterWorkerCapability.js

```diff
@@ -4,12 +4,14 @@ var TaskRouterCapability = require('./TaskRouterCapability');
 function TaskRouterWorkerCapability(accountSid, authToken, workspaceSid, workerSid) {
     TaskRouterCapability.call(this, accountSid, authToken, workspaceSid, workerSid);
 
-    this.reservationsUrl = this._baseUrl + "/Tasks/**";
+    this.tasksUrl = this._baseUrl + "/Tasks/**";
```

Comment:
same here, if I had an instance of `TaskRouterWorkerCapbility` that accessed `tasksUrl`, it would break with this update.